### DISCUSSION TITLE
Missing Panels

### DISCUSSION THREAD

**Discussion by Speedwagen** [blackears/cyclopsLevelBuilder#110] [Q&A]
I am missing the Faces panel and the Tool Properties is blank
![image](https://github.com/blackears/cyclopsLevelBuilder/assets/110987088/95f18607-e87f-4c2a-bcc9-059df7344a26)
![image](https://github.com/blackears/cyclopsLevelBuilder/assets/110987088/94779546-68a3-460d-bc0f-314d38809133)

**Comment by Speedwagen** [comment]
ok, I have semi-fixed it by grabbing the files from the master repo. Now after upgrading the main Cyclops object the new building block object does not have a script attached and I am getting parse errors
`  res://addons/cyclops_level_builder/tools/tool_block.gd:129 - Parse Error: Could not find type "CommandAddBlock" in the current scope.
  res://addons/cyclops_level_builder/tools/tool_block.gd:129 - Parse Error: Identifier "CommandAddBlock" not declared in the current scope.
  res://addons/cyclops_level_builder/cyclops_level_builder.gd:-1 - Compile Error: 
  modules/gdscript/gdscript.cpp:2775 - Failed to load script "res://addons/cyclops_level_builder/cyclops_level_builder.gd" with error "Compilation failed". (User)
Add Autoload
  res://addons/cyclops_level_builder/cyclops_level_builder.gd:125 - Invalid call. Nonexistent function 'new' in base 'GDScript'.
  res://Assets/Goblin/goblin.gd:2 - Parse Error: Class "Goblin" hides a global script class.
  modules/gdscript/gdscript.cpp:2775 - Failed to load script "res://Assets/Goblin/goblin.gd" with error "Parse error". (User)`

**Comment by blackears** [comment]
I don't know.  Some of the errors make it look like you may have class name conflicts, so make sure that none of your classes use the same class names as classes in the project.  GDScript doesn't support namespaces yet, so name conflicts are hard to avoid.  You can also try creating a new project and then adding your current project into it in pieces until you find out what caused it to break.
